fuel/schemes.py

```diff
@@ -95,13 +104,26 @@ class ConcatenatedScheme(IterationScheme):
         A list of :class:`IterationSchemes`, whose request iterators
         are to be concatenated in the order given.
 
+    Notes
+    -----
+    All schemes must produce the same type of requests (batches or
```

Comment:
I think a better way of phrasing it is that "Each scheme must produce either batches or examples, but not a mix of both."